README.md

```diff
@@ -95,11 +117,6 @@ You can pass in props to define tip direction, styling, etc.  Content is the onl
       <td>string</td>
       <td>padding amount for the tooltip contents (defaults to '10px')</td>
     </tr>
-    <tr>
```

Comment:
This prop was for the target *wrapper*, not the tooltip bubble. Since it's now updated to not render the wrapper itself but actually require it as a child, it is obsolete.